src/main/resources/data/crossroads/patchouli_books/manual/zh_cn/entries/intro/heat.json

```diff
@@ -26,7 +26,7 @@
 		},
 		{
 			"type": "text",
-			"text": "<bobo>要轴快点了！！！/$$(br2)"
+			"text": "<bobo>g0TTa g0 FAST！！！/$$(br2)"
```

Comment:
Why is this translation reverted to English?

Reply:
I realized it is a sonic reference and decided to revert it so that the origin can be found by using search engine.